addons/l10n_be/data/template/account.tax-be.csv

```diff
@@ -197,6 +197,11 @@
 "","","","","","","","","","","","base","refund","+85||-82","","","","","",""
 "","","","","","","","","","","","tax","refund","-82||+85","","65","","","",""
 "","","","","","","","","","","","tax","refund","+63","a451","35","","","",""
+"attn_VAT-IN-21-D35","735","21% IG D35","21%","21% IG D35","21.0","percent","purchase","tax_group_tva_21","invest","","base","invoice","+83","","","","","fiscal_position_template_1,fiscal_position_template_5",""
```

Comment:
Might want to specify either `V83`, which is the grid for investment goods, or `IG` in the external ID? @casp-odoo

Reply:
V83